3rdParty/LightPcapNg/LightPcapNg/src/light_pcapng.c

```diff
@@ -182,7 +181,7 @@ void parse_by_block_type(struct _light_pcapng *current, const uint32_t *local_da
       {
          DPRINT_HERE(LIGHT_CUSTOM_DATA_BLOCK);
          struct _light_custom_nonstandard_block *cnb = NULL;
-         struct _light_option *opt = NULL;
+         struct _light_option *opt = calloc(1, sizeof(struct _light_option));
```

Comment:
Ditto: memory leak

Reply:
Yes done some Improvement to memory free(opt) now there wouldn't be any leakage i think